### DISCUSSION TITLE
Creating a cask for a CLI tool

### DISCUSSION THREAD

**Discussion by aaditkamat** [Homebrew/discussions#2260] [Writing Formulae/Casks] [answered]
Hi everyone,

I have contributed a [cask for the Surfshark app on MacOS before.](https://github.com/Homebrew/homebrew-cask/pull/86512) This time however I wish to contribute a cask for the [TagUI project](https://github.com/kelaberetiv/TagUI) which can be installed as a CLI. May I know which cask(s) I should refer to before I can create my own?

**Comment by SMillerDev** [comment, marked as answer]
If it's an open source CLI it won't be accepted as a cask but should be a formulae instead.
